jobs-shell/script.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START run_jobs_shell_script]
+set -euox pipefail
+
+echo "Hello from a bash script!"
```

Comment:
This sample code should match the [canonical](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/main/run/jobs)